lib/configuration/sthConfiguration.js

```diff
@@ -30,673 +30,731 @@ var sthLogger = require('logops');
-    comp: 'STH',
-    op: module.exports.OPERATION_TYPE.STARTUP
-  },
-  SHUTDOWN: {
-    from: 'n/a',
-    srv: 'n/a',
-    subsrv: 'n/a',
-    comp: 'STH',
-    op: module.exports.OPERATION_TYPE.SHUTDOWN
-  },
-  SERVER_START: {
-    from: 'n/a',
-    srv: 'n/a',
-    subsrv: 'n/a',
-    comp: 'STH',
-    op: module.exports.OPERATION_TYPE.SERVER_START
-  },
-  SERVER_STOP: {
-    from: 'n/a',
-    srv: 'n/a',
-    subsrv: 'n/a',
-    comp: 'STH',
-    op: module.exports.OPERATION_TYPE.SERVER_STOP
-  },
-  SERVER_LOG: {
-    from: 'n/a',
-    srv: 'n/a',
-    subsrv: 'n/a',
-    comp: 'STH',
-    op: module.exports.OPERATION_TYPE.SERVER_LOG
-  },
-  DB_CONN_OPEN: {
-    from: 'n/a',
-    srv: 'n/a',
-    subsrv: 'n/a',
-    comp: 'STH',
-    op: module.exports.OPERATION_TYPE.DB_CONN_OPEN
-  },
-  DB_CONN_CLOSE: {
-    from: 'n/a',
-    srv: 'n/a',
-    subsrv: 'n/a',
-    comp: 'STH',
-    op: module.exports.OPERATION_TYPE.DB_CONN_CLOSE
-  },
-  DB_LOG: {
-    from: 'n/a',
-    srv: 'n/a',
-    subsrv: 'n/a',
-    comp: 'STH',
-    op: module.exports.OPERATION_TYPE.DB_LOG
-  }
+    STARTUP: {
+        from: 'n/a',
+        srv: 'n/a',
+        subsrv: 'n/a',
+        comp: 'STH',
+        op: module.exports.OPERATION_TYPE.STARTUP
+    },
+    SHUTDOWN: {
+        from: 'n/a',
+        srv: 'n/a',
+        subsrv: 'n/a',
+        comp: 'STH',
+        op: module.exports.OPERATION_TYPE.SHUTDOWN
+    },
+    SERVER_START: {
+        from: 'n/a',
+        srv: 'n/a',
+        subsrv: 'n/a',
+        comp: 'STH',
+        op: module.exports.OPERATION_TYPE.SERVER_START
+    },
+    SERVER_STOP: {
+        from: 'n/a',
+        srv: 'n/a',
+        subsrv: 'n/a',
+        comp: 'STH',
+        op: module.exports.OPERATION_TYPE.SERVER_STOP
+    },
+    SERVER_LOG: {
+        from: 'n/a',
+        srv: 'n/a',
+        subsrv: 'n/a',
+        comp: 'STH',
+        op: module.exports.OPERATION_TYPE.SERVER_LOG
+    },
+    DB_CONN_OPEN: {
+        from: 'n/a',
+        srv: 'n/a',
+        subsrv: 'n/a',
+        comp: 'STH',
+        op: module.exports.OPERATION_TYPE.DB_CONN_OPEN
+    },
+    DB_CONN_CLOSE: {
+        from: 'n/a',
+        srv: 'n/a',
+        subsrv: 'n/a',
+        comp: 'STH',
+        op: module.exports.OPERATION_TYPE.DB_CONN_CLOSE
+    },
+    DB_LOG: {
+        from: 'n/a',
+        srv: 'n/a',
+        subsrv: 'n/a',
+        comp: 'STH',
+        op: module.exports.OPERATION_TYPE.DB_LOG
+    }
 };
 
 var invalidLoggingLevel;
 var LOGGING_LEVELS = ['DEBUG', 'INFO', 'WARN', 'ERROR', 'FATAL'];
 if (ENV.LOGOPS_LEVEL && LOGGING_LEVELS.indexOf(ENV.LOGOPS_LEVEL.toUpperCase()) !== -1) {
-  module.exports.LOGOPS_LEVEL = ENV.LOGOPS_LEVEL.toUpperCase();
-} else if (config && config.logging && config.logging.level &&
-  LOGGING_LEVELS.indexOf(config.logging.level.toUpperCase()) !== -1) {
-  module.exports.LOGOPS_LEVEL = config.logging.level.toUpperCase();
+    module.exports.LOGOPS_LEVEL = ENV.LOGOPS_LEVEL.toUpperCase();
+} else if (
```

Comment:
This (and other similar ones) would be clearer with at least all the `config` existence clauses checking in the same line, i.e.

```
} else if (config && config.logging && config.loginglevel && ...)
```

Reply:
Fixed  8b8777

Reply:
In 8b8777d I see several fixes of this but it seems the ones in L138 and L156 are still there...

Reply:
... and L248

Going one by one can be hard and error prone... doesn't allow prettier to make a general exception for this?

Reply:
Fixed  c00464eb , but no - the whole point about prettier is that is it **opinionated** not **configurable**.